addons/api/mirage/route-handlers/target.js

```diff
@@ -9,8 +9,8 @@ export function targetHandler({ targets }, { params: { idMethod } }) {
   const method = idMethod.split(':')[1];
   const target = targets.find(id);
   const {
-    brokeredCredentialSourceIds: originalBrokeredCredentials,
-    injectedApplicationCredentialSourceIds: originalInjectedCredentials,
+    brokeredCredentialSourceIds: originalBrokeredCredentials = [],
+    injectedApplicationCredentialSourceIds: originalInjectedCredentials = [],
```

Comment:
added default value of empty array because when undefined it causes an error since on line 26 the spread operator is used.

Reply:
Was this just broken before?

Reply:
yup but only when creating a new target in mirage and trying to add cred sources.